pkgs/by-name/do/dotnet-outdated/package.nix

```diff
@@ -1,42 +1,21 @@
 { lib
-, fetchFromGitHub
-, buildDotnetModule
+, buildDotnetGlobalTool
 , dotnetCorePackages
 }:
-let
-  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_6_0 sdk_7_0 sdk_8_0 ];
-
-in buildDotnetModule rec {
-  pname = "dotnet-outdated";
+buildDotnetGlobalTool {
+  pname = "dotnet-outdated-tool";
```

Comment:
`pname` shouldn't change imo

Reply:
Yup agreed - I changed it because the nuget is called `dotnet-outdated-tool`, but `buildDotnetGlobalTool` has a `nugetName` param which I have used instead now.